x/bundles/keeper/keeper_suite_valid_bundles_test.go

```diff
@@ -245,7 +245,7 @@ var _ = Describe("valid bundles", Ordered, func() {
 		// assert payout transfer
 		Expect(balanceUploader).To(Equal(initialBalanceStaker0))
 		// assert commission rewards
-		Expect(uploader.CommissionRewards).To(Equal(uploaderPayoutReward + storageReward))
+		Expect(uploader.CommissionRewards.AmountOf(globalTypes.Denom).Uint64()).To(Equal(uploaderPayoutReward + storageReward))
```

Comment:
low prio: Should we maybe turn all these lines to
Expect(uploader.CommissionRewards.String()).To(Equal(sdk.NewInt64Coin(globalTypes.Denom, int64(uploaderPayoutReward+storageReward)).String()))

to check that other denoms are zero

Reply:
Actually doing this right now in separate PR on `troy/multiple-coins-integration-tests`